contracts/InterchainTokenService.sol

```diff
@@ -893,20 +855,19 @@ contract InterchainTokenService is
         string memory originalSourceChain = sourceChain;
```

Comment:
```suggestion
        string memory originalSourceChain;
```
doesn't need to be initialized to sourceChain anymore

Reply:
nice catch.